package.json

```diff
@@ -29,6 +29,7 @@
     "@babel/runtime": "^7.5.5",
     "axios": "^0.19.0",
     "babel-eslint": "^10.0.2",
+    "bcrypt": "^3.0.6",
```

Comment:
why bcrypt?

Reply:
are you hashing the password on the front end or there is the other reason you are using bycript